tests/actions.go

```diff
@@ -228,10 +248,26 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterInfo) error {
 		}
 	}
 
+
+	//for the test should add two clusters in a namespace, so we
+	//has to use clustername as an label.
+	sets := map[string]string {
+		label.InstanceLabelKey:		info.ClusterName,
+	}
+	var buffer bytes.Buffer
+	for k,v := range sets {
+		set := fmt.Sprintf("-l %s=%s",k , v)
```

Comment:
let sets = {"foo": "a", "bar": "b"}, you finally get a buffer with `"-l foo=a-l bar=b"`

Reply:
there is only one pair in this field , I make some change .thx